### PR TITLE
Template move/delete templates from sdk

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

**Comment by jmarolf** [comment]
@nguerrera please review

**Comment by nguerrera** [comment]
> +0 −1,707

LGTM

**Comment by nguerrera** [comment]
Ship it when green and when VSO build has been validated.

File: src/VsixV3/Microsoft.VisualStudio.Templates.CSharp.NetCore.vsmanproj [hunk 1]
```diff
@@ -16,16 +16,12 @@
   <Import Project="$(NuGet_Packages)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.targets" />
 
   <Target Name="BeforeBuild">
-    <ItemGroup>
-      <TemplatesVsixFilesToCopy Include="$(OutDir)\Microsoft.VisualStudio.Templates.CSharp.NetCore.json;$(OutDir)\Microsoft.VisualStudio.ProjectSystem.CSharp.Templates.vsix" />
-    </ItemGroup>
     <Copy SourceFiles="@(TemplatesVsixFilesToCopy)" DestinationFolder="$(OutputPath)" />
```
Comment (srivatsn) [inline src/VsixV3/Microsoft.VisualStudio.Templates.CSharp.NetCore.vsmanproj]: You'll need to delete this line.

**Review by srivatsn — APPROVED** [review]

**Review by nguerrera — APPROVED** [review]

**Pull request merged**